.gitignore

```diff
@@ -0,0 +1,19 @@
+.svn/
+.hg/
+.hgtags/
+log
+*.py[cod]
+build
+doc/_build
+dist/
+pylint.egg-info/
+.tox
+*.sw[a-z]
+doc/features.rst
+pyve
+build-stamp
+debian/files
```

Comment:
Are you sure that is correct? It looks to me that in the existing `.hgignore`, there several [rooted](https://www.selenic.com/mercurial/hgignore.5.html) patterns (starting with `^`). But your `.gitignore` has no rooted patterns (starting with `/`).